Makefile

```diff
@@ -15,9 +15,9 @@ MANIFEST_PATH=./manifests/btp-operator
 MANIFEST_FILE=btp-manager.yaml
 
 # Image URL to use all building/pushing image targets
-IMG_REGISTRY_PORT ?= 5001
-IMG_REGISTRY ?= k3d-kyma-registry:$(IMG_REGISTRY_PORT)
-IMG ?= $(IMG_REGISTRY)/btp-manager:$(MODULE_VERSION)
+IMG_REGISTRY_PORT ?= 5000
```

Comment:
As far as I remember in the scripts both ports were used deliberately. Port 5000 and 5001.